### PR TITLE
Add link to get auth transaction

### PR DESCRIPTION
**Pull request by dara123**

### CODE REVIEW

**Comment by dara123** [comment]
devs please review

File: CHANGELOG.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ _TBD_
 - [x] Added new properties to `Customer`: `averageValue`, `paymentCount`, `lastPaymentTime`
 - [x] Added new service `WebhookCredentials`
 - [x] Added new property to `PaymentCardAuthorization`: `redirectUrl` 
-- [x] Added new link to `PaymentCard`: `approvalUrl` 
+- [x] Added new link to `PaymentCard`: `approvalUrl`, `authorizedTransaction`
```
Comment (slavcodev) [inline CHANGELOG.md]: I think I saw `authTransaction` in API response.

Comment (slavcodev) [inline CHANGELOG.md]:
```suggestion
- [x] Added new link to `PaymentCard`: `approvalUrl`, `authTransaction` 
```

File: src/Entities/PaymentCard.php [hunk 2]
```diff
@@ -265,4 +265,13 @@ public function getApprovalUrl()
     {
         return $this->getLink('approvalUrl');
     }
+
+    public function getAuthorizedTransaction()
```
Comment (slavcodev) [inline src/Entities/PaymentCard.php]: I wonder if the inconsistency between link and getter might be confused.

**Pull request merged**